service-api/module/Application/src/Model/IdMethod.php

```diff
@@ -19,4 +19,8 @@ enum IdMethod: string
     case PostOfficeWithEUDrivingLicense = "po_eud";
     case PostOfficeWithInternationalDrivingLicence = "po_ind";
     case PostOfficeWithNoneOfTheAbove = "po_n";
+    case IntlPassport = 'xpn';
+    case PhotocardDrivingLicence = 'xdln';
+    case NationalIdentityCard = 'xid';
+    case EUId = 'euid';
```

Comment:
I'm a bit confused how these fit in with values like `PostOffice` and `PostOfficeWithInternationalPassport`. Maybe this enum needs to be split up into something like "the route the customer has chosen", "the type of ID they're providing" and "the type of country they're providing from (UK/EU/international)"?

Reply:
Agreed - the id method now stores different types of data at different points in the case journey - eg it stores the post-office ID route as an alternative to the on-the-phone id documents, but then changes this to the eventual id document to be presented at the post office. This needs refactoring, but ideally in a new ticket because it will break other parts of the system